packages/repository/src/__tests__/unit/repositories/relation.helpers.unit.ts

```diff
@@ -3,113 +3,398 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {expect} from '@loopback/testlab';
-import {DefaultCrudRepository, findByForeignKeys, juggler} from '../../..';
+import {expect, toJSON} from '@loopback/testlab';
+import {
+  DefaultCrudRepository,
+  findByForeignKeys,
+  includeRelatedModels,
+  juggler,
+} from '../../..';
 import {model, property} from '../../../decorators';
 import {Entity} from '../../../model';
+import {
+  belongsTo,
+  BelongsToAccessor,
+  Getter,
+  hasMany,
+  HasManyRepositoryFactory,
+  InclusionResolver,
+} from '../../../relations';
 
-describe('findByForeignKeys', () => {
+describe('relation helpers', () => {
```

Comment:
This test file is growing quite large. Can we split it into two files please? One test file for `findByForeignKeys` (this can be hopefully just a rename from `relation.helpers.unit.ts` to something like `find-by-foreing-key.ts` or `relations-helpers/find-by-foreign.keys`, with no code changes needed), another test file for the new helper `includeRelatedModels`.